launcher/src/main/go/go.sum

```diff
@@ -0,0 +1,545 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
This seems like way too many dependencies

Reply:
Cleaned up.